bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/api/ShellyApiJsonDTO.java

```diff
@@ -469,6 +502,11 @@
         @SerializedName("sleep_mode")
         public ShellySensorSleepMode sleepMode; // FW 1.6
 
+        // @SerializedName("ext_temperature")
+        // public ShellyStatusSensor.ShellyExtTemperature extTemperature; // Shelly 1/1PM: sensor values
+        // @SerializedName("ext_humidity")
+        // public ShellyStatusSensor.ShellyExtHumidity extHumidity; // Shelly 1/1PM: sensor values
+
```

Comment:
in general yes, those are existing, but code doesn't use them at the moment. It makes it easier in case of, because making the comparisons and adding new ones is always a pita
if you request I could remove them, there are a lot more for the same reason

Reply:
I think it's better then, if you remove the comments and make it normal code. It can be refactored, is scanned by the static code analysis and so on.